### ISSUE TITLE
"fatal: Cannot handle files this big"

### ISSUE THREAD

**Issue by ElectroBandit** [Acris/shadowsocks-asuswrt-merlin#59]
During installation on the RT-AX58U, an error occurs:

```
Cloning shadowsocks-asuswrt-merlin...
Cloning into '/opt/share/ss-merlin'...
fatal: Cannot handle files this big
 Error: git clone of shadowsocks-asuswrt-merlin repo failed
```

**Comment by Angel-1024** [comment]
Same on RT-AX82U

**Comment by ElectroBandit** [comment]
I also tried to install it on RT-AX56U and everything went ok. Just took the files from AX56U and manually copied them to the AX58U same entware directories. Then I launched a slightly modified "install.sh", and now it works on AX58U.

**Comment by Antonio1297** [comment]
Same error on rt-ac68u.
How to solve this problem?

**Comment by Antonio1297** [comment]
> Hi, can you run `df -h` command on your router and paste the outputs here?

admin@RT-AC68U-9F00:/tmp/home/root# df -h
Filesystem                Size      Used Available Use% Mounted on
/dev/root                33.0M     33.0M         0 100% /
devtmpfs                124.8M         0    124.8M   0% /dev
tmpfs                   124.9M      1.6M    123.2M   1% /tmp
/dev/mtdblock4           62.8M      2.6M     60.2M   4% /jffs
/dev/sda1                 1.8G     70.3M      1.7G   4% /tmp/mnt/sda1

**Comment by Antonio1297** [comment]
> Hi, please run the installation command again and check if the issue has been fixed.

The same error (

**Comment by Antonio1297** [comment]
I manually copied all the folders, then ran install.sh (corrected). Everything installed. But when I run ss-merlin start, I get the error ss-redir: Error loading shared libraries: libsodium.so.23.

**Comment by Antonio1297** [comment]
> > I manually copied all the folders, then ran install.sh (corrected). Everything installed. But when I run ss-merlin start, I get the error ss-redir: Error loading shared libraries: libsodium.so.23.
> 
> Which version of Asuswrt-Merlin are you using?

386.14.0

**Comment by Antonio1297** [comment]
I already did this. No results.
I installed entware with amtm. Maybe that's the reason?..

**Comment by ElectroBandit** [comment]
> I installed entware with amtm. Maybe that's the reason?..

No. I installed it on AX56U, AX58U and AX86U PRO same way (entware with amtm, same Asuswrt-Merlin version). And only AX58U have this issue. After manually copying files i also have "ss-redir: Error loading shared libraries: libsodium.so.23". I also manually copied libsodium.so.23 directories from AX56U to AX58U, and then it worked.

**Comment by k-g-g** [comment]
Hi!
This this some troubles with last git_2.45.2.  
Try to install older version git_2.39.2 from archive and skip  "opkg upgrade" in install.sh
Checked on R7000

**Comment by ageevvalentin** [comment]
I fixed the same issues after disabling the "Parallel use of Entware backports repo" in ampt -> ep. On AC-87U

**Comment by Antonio1297** [comment]
> I fixed the same issues (libsodium and git) after disabling the "Parallel use of Entware backports repo" in amtm -> ep immediately after installing Entware. On AC-87U

Cheers! That's work.
